Source/WebCore/html/OffscreenCanvas.h

```diff
@@ -158,6 +158,7 @@ class OffscreenCanvas final : public ActiveDOMObject, public RefCounted<Offscree
     void dispatchEvent(Event&) final;
     using RefCounted::ref;
     using RefCounted::deref;
+    bool isDetached() const { return m_detached; };
```

Comment:
@cdumez @mattwoodrow this is the only non-WebGPU change. We need to know if an OffscreenCanvas is detached to pass some of the tests.

I'm not aware of an alternative method of determining that other than adding a new member function on OffscreenCanvas to return that internal state variable.